src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -1,31 +1,36 @@
 import * as React from 'react';
-import { Text } from 'react-native';
+import {Text} from 'react-native';
 
-import { colors } from "../../../utils/colors"
-import { fonts } from "../../../utils/fonts"
+import {fonts} from '../../../utils/fonts'
+import {COLORS} from '../../../utils/theme';
 
 /**
  * @typedef {Object} DiscoveryTitleSeparatorProp
  * @property {String} text
  */
 /**
- * 
- * @param {DiscoveryTitleSeparatorProp} prop 
+ *
+ * @param {DiscoveryTitleSeparatorProp} prop
  */
 const DiscoveryTitleSeparator = (prop) => {
-    const { text } = prop
-    return <Text style={{
+  const {text} = prop;
+  return (
+    <Text
+      style={{
         fontSize: 12,
         lineHeight: 18,
-        fontFamily : fonts.poppins[600],
-        color: colors.black,
+        fontFamily: fonts.poppins[600],
+        color: COLORS.black,
         paddingLeft: 20,
         paddingRight: 20,
         paddingTop: 11,
         paddingBottom: 11,
-        backgroundColor: colors.lightgrey,
+        backgroundColor: COLORS.lightgrey
         // marginTop: 15,
-    }}>{text}</Text>
-}
+      }}>
+      {text}
+    </Text>
+  );
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The file should end with a newline to comply with the POSIX standard, which states that a line is a sequence of zero or more non-newline characters followed by a newline character. Not having a newline at the end of the file can cause issues with some tools.
```diff
- export default DiscoveryTitleSeparator;
+ export default DiscoveryTitleSeparator;\n
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->